src/PerfView/Extensibility.cs

```diff
@@ -994,6 +994,36 @@ public void LookupSymbolsForModule(string simpleModuleName, SymbolReaderOptions
             }
         }
 
+        /// <summary>
+        /// Saves the stacks as a CSV in the same format as it would appear in the CPUStacks GUI
+        /// </summary>
+        /// <param name="outputFileName"> The file name the data will be written to </param>
+        public void SaveAsCsvByName(string outputFileName)
+        {
```

Comment:
NIT: Might be worth adding a precondition check on the outputFileName being null / empty.

```csharp
  if (string.IsNullOrEmpty(fileName))
  {
      throw new ArgumentException($"{nameof(fileName)} is null or empty.");
  }